src/Twig/CommonExtension.php

```diff
@@ -73,9 +77,9 @@ private function getLocale($item): ?string
     {
         if (is_string($item)) {
             $localepattern = '/^[a-z]{2}((-|_)[a-z]{2})?$/m';
-            preg_match_all($localepattern, $item, $matches);
+            $matchCount = preg_match_all($localepattern, $item);
 
-            return ! empty($matches) ? $item : null;
+            return $matchCount > 0 ? $item : null;
```

Comment:
Would the usage of just `preg_match` not be enough here?

Reply:
Sure, I fix it